app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java

```diff
@@ -162,10 +153,43 @@ private void deleteConfirmation(int position) {
         deleteIndex = position;
     }
 
+    // Handles when a user clicked on the button to show a quiz
+    //    private void showQuiz(int position) { // TODO next sprint
+    //        final String quizID = quizzes.get(position).getKey();
+    //        progressBar.setVisibility(VISIBLE);
+    //
+    //        CompletableFuture<StringPool> stringPool =
+    //                db.getQuizLanguages(quizID)
+    //                        .thenCompose(
+    //                                languages ->
+    //                                        db.getQuizStringPool(quizID,
+    // getBestLanguage(languages)));
+    //
+    //        CompletableFuture<Quiz> quizStructure = db.getQuizStructure(quizID);
+    //
+    //        CompletableFuture.allOf(stringPool, quizStructure)
+    //                .whenComplete(
+    //                        (aVoid, throwable) -> {
+    //                            if (throwable != null)
+    //                                Toast.makeText(
+    //                                                requireContext(),
+    //                                                R.string.database_error,
+    //                                                Toast.LENGTH_SHORT)
+    //                                        .show();
+    //                            else
+    //                                launchQuizActivity(
+    //                                        quizStructure
+    //                                                .join()
+    //                                                .instantiateLanguage(stringPool.join()));
+    //                        });
+    //
+    //        progressBar.setVisibility(INVISIBLE);
+    //    }
+
```

Comment:
Why is all of this commented?

Reply:
This is just temporary, the time constraints did not allow me to integrate this into the workflow. So I couldn't test it and in order to not reduce the coverage unnecessarily I commented this part.